scripts/gods/god.gd

```diff
@@ -0,0 +1,28 @@
+class_name God
+extends Node2D
```

Comment:
Should inherit from Node rather than Node 2D

Reply:
Changed.